vendor/bat-native-ads/src/bat/ads/internal/server/redeem_unblinded_token/create_confirmation_util.cc

```diff
@@ -23,43 +27,61 @@ using challenge_bypass_ristretto::TokenPreimage;
 
 std::string CreateConfirmationRequestDTO(
     const ConfirmationInfo& confirmation) {
-  base::Value payload(base::Value::Type::DICTIONARY);
+  base::Value dto(base::Value::Type::DICTIONARY);
 
-  payload.SetKey("creativeInstanceId",
+  dto.SetKey("creativeInstanceId",
       base::Value(confirmation.creative_instance_id));
 
-  payload.SetKey("payload", base::Value(base::Value::Type::DICTIONARY));
+  dto.SetKey("payload", base::Value(base::Value::Type::DICTIONARY));
 
   const std::string blinded_payment_token_base64 =
       confirmation.blinded_payment_token.encode_base64();
-  payload.SetKey("blindedPaymentToken",
+  dto.SetKey("blindedPaymentToken",
       base::Value(blinded_payment_token_base64));
 
   const std::string type = std::string(confirmation.type);
-  payload.SetKey("type", base::Value(type));
+  dto.SetKey("type", base::Value(type));
 
   DCHECK(!_build_channel.name.empty());
-  payload.SetKey("buildChannel", base::Value(_build_channel.name));
+  dto.SetKey("buildChannel", base::Value(_build_channel.name));
 
   if (_build_channel.is_release) {
     const std::string locale =
         brave_l10n::LocaleHelper::GetInstance()->GetLocale();
 
     if (locale::HasLargeAnonymity(locale)) {
       const std::string country_code = brave_l10n::GetCountryCode(locale);
-      payload.SetKey("countryCode", base::Value(country_code));
+      dto.SetKey("countryCode", base::Value(country_code));
+
+      experiments::PageProbabilities page_probabilities_experiment;
```

Comment:
wondering how this would work for future experiments? Just adding them 1 by 1?

Reply:
for now we only allow one simultaneous experiment within ads (the seed generator only ever generates one for now). In the future we can iterate over an experiments container and build the dict dynamically.